MantidQt/CustomInterfaces/src/Tomography/TomographyIfacePresenter.cpp

```diff
@@ -651,17 +668,10 @@ bool TomographyIfacePresenter::isLocalResourceSelected() const {
 }
 
 void TomographyIfacePresenter::processRefreshJobs() {
-  // No need to be logged in, there can be local processes
-  if (m_model->loggedIn().empty()) {
-    m_model->doRefreshJobsInfo("Local");
-    m_view->updateJobsInfoDisplay(m_model->jobsStatus(),
-                                  m_model->jobsStatusLocal());
-    return;
-  }
-
   m_model->doRefreshJobsInfo(m_view->currentComputeResource());
 
   {
+    // BUG :: still crash here if closed during running process
```

Comment:
Does this bug still exist - I'm not a fan of leaving comments like this as they usually get forgotten about when the bug is fixed. Instead you could link to the line number in the issue.